data/han2020sequential/han2020sequential-extraction.md

```diff
@@ -0,0 +1,154 @@
+        "biomarker": "SARS-CoV-2",
+        "limit_of_quantification": "unknown",
+        "limit_of_detection": limit_of_detection,
+        "unit": "gc/mL",
+        "reference_event": "symptom onset"
+    },
+    "oropharynx_E": {
+        "description": "SARS-CoV-2 RNA gene copy concentration in oropharynx samples.",
+        "specimen": "oropharyngeal_swab",
+        "biomarker": "SARS-CoV-2",
+        "limit_of_quantification": "unknown",
+        "limit_of_detection": limit_of_detection,
+        "unit": "gc/mL",
+        "reference_event": "symptom onset"
+    },
+    "NPSOPS_E": {
+        "description": "SARS-CoV-2 RNA gene copy concentration in oropharynx and "
+        "nasopharynx samples.",
+        "specimen": ["nasopharyngeal_swab", "oropharyngeal_swab"],
+        "biomarker": "SARS-CoV-2",
+        "limit_of_quantification": "unknown",
+        "limit_of_detection": limit_of_detection,
+        "unit": "gc/mL",
+        "reference_event": "symptom onset"
+    },
+    "plasma_E": {
+        "description": "SARS-CoV-2 RNA gene copy concentration in plasma samples.",
+        "specimen": "plasma",
+        "biomarker": "SARS-CoV-2",
+        "limit_of_quantification": "unknown",
+        "limit_of_detection": limit_of_detection,
+        "unit": "gc/mL",
+        "reference_event": "symptom onset"
+    },
+    "saliva_E": {
+        "description": "SARS-CoV-2 RNA gene copy concentration in saliva samples.",
+        "specimen": "saliva",
+        "biomarker": "SARS-CoV-2",
+        "limit_of_quantification": "unknown",
+        "limit_of_detection": limit_of_detection,
+        "unit": "gc/mL",
+        "reference_event": "symptom onset"
+    },
+    "urine_E": {
+        "description": "SARS-CoV-2 RNA gene copy concentration in urine samples.",
+        "specimen": "urine",
+        "biomarker": "SARS-CoV-2",
+        "limit_of_quantification": "unknown",
+        "limit_of_detection": limit_of_detection,
+        "unit": "gc/mL",
+        "reference_event": "symptom onset"
+    },
+    "sputum_E": {
+        "description": "SARS-CoV-2 RNA gene copy concentration in sputum samples.",
+        "specimen": "sputum",
+        "biomarker": "SARS-CoV-2",
+        "limit_of_quantification": "unknown",
+        "limit_of_detection": limit_of_detection,
+        "unit": "gc/mL",
+        "reference_event": "symptom onset"
+    },
+    "stool_E": {
+        "description": "From manuscript: \"Viral RNA was detected using the PowerChek "
+        "2019-nCoV real-time polymerase chain reaction kit (Kogene Biotech, Seoul, "
+        "Korea) for amplification of the E gene and the RNA-dependent RNA polymerase "
+        "region of the ORF1b gene, and quantified with a standard curve that was "
+        "constructed using in vitro transcribed RNA provided from the European Virus "
+        "Archive.\" However, data reported in Figure 1 explicitly refer to the E gene "
+        "target.",
+        "specimen": "sputum",
```

Comment:
The `specimen` should be "stool" here.
This one does have the `gene_target`. But based on the final YAML file. It should be after `specimen` instead of after `limit_of_detection`.